### PR TITLE
Add 2 Mathematics Books

### PR DESCRIPTION
**Pull request by fabiohenriquecn**
I could add the authors too.
The Wisconsin book is an open source project.

### CODE REVIEW

**Comment by eshellman** [comment]
yes, please add authors!

**Comment by eshellman** [comment]
Also thanks for restarting this repo. @onebree got me to create it, but it sort of died off. There are now lots of open textbooks that might be added. The visibility is now higher as we've taken on admin of the free-programming-books repo.

**Comment by fabiohenriquecn** [comment]
I've added the year.

File: free-science-books.md [hunk 1]
```diff
@@ -3,7 +3,9 @@
 # Science (General) (Q)
 
 ## Mathematics (QA)
+* [Elementary Algebra](http://www.mathhands.com/046), by Francisco Javier Márquez (2011).
 * [Stacks Project](http://stacks.math.columbia.edu/)
+* [Wisconsin Calculus](https://www.math.wisc.edu/undergraduate/calcstudentpage), by Sigurd B. Angenent et al. (2015).
```
Comment (eshellman) [inline free-science-books.md]: This is pretty far from being a book.

Comment (fabiohenriquecn) [inline free-science-books.md]: I'll remove that.

**Pull request merged**